js/lex.go

```diff
@@ -289,35 +288,35 @@ func (l *Lexer) consumeUnicodeEscape() bool {
 	return true
 }
 
-////////////////////////////////////////////////////////////////
-
-func (l *Lexer) consumeSingleLineToken(skip int) {
-	l.r.Move(skip)
```

Comment:
Curious - why did you move this out? Seems like a nice abstraction given that you always need to skip some prefix when consuming a line comment.